pkg/network/ebpf/c/protocols/redis/types.h

```diff
@@ -0,0 +1,19 @@
+#ifndef __REDIS_TYPES_H
+#define __REDIS_TYPES_H
+
+#include "conn_tuple.h"
+
+// Controls the number of Redis transactions read from userspace at a time.
+#define REDIS_BATCH_SIZE 25
```

Comment:
I'd start with the highest value possible, so we can always ensure we're maximizing the batches functionality